bin/handler-xmpp.rb

```diff
@@ -4,9 +4,10 @@
 require 'sensu-handler'
 require 'xmpp4r/client'
 require 'xmpp4r/muc'
-include Jabber
 
 class XmppHandler < Sensu::Handler
+  include Jabber
```

Comment:
hmm could I see the rubocop rule that said you should move it here?  I can't think of why this needs to be moved.